build/shade/_k-standard-goals.shade

```diff
@@ -219,6 +219,10 @@ default PACKAGELIST_JSON_FILENAME = 'NuGetPackageVerifier.json'
         var project = (JsonObject)Json.Deserialize(projectText);
         var configs = project.ValueAsJsonObject("frameworks");
         var targetFrameworks = configs == null ? new string[0] : configs.Keys;
+        if (!string.IsNullOrEmpty(BuildFramework))
+        {
+            targetFrameworks = targetFrameworks.Where(t => t.Equals(BuildFramework, StringComparison.OrdinalIgnoreCase)).ToList();
```

Comment:
No, `string` is not the problem. The error will be something like "Cannot assign `IEnumerable<string>` to `ICollection<string>`".